src/error-handlers/type.js

```diff
@@ -6,25 +6,73 @@ import * as Instance from "@hyperjump/json-schema/instance/experimental";
  * @import { ErrorHandler, ErrorObject } from "../index.d.ts"
  */
 
+const ALL_TYPES = ["null", "boolean", "number", "string", "array", "object", "integer"];
+
 /** @type ErrorHandler */
 const type = async (normalizedErrors, instance, localization) => {
   /** @type ErrorObject[] */
   const errors = [];
 
   if (normalizedErrors["https://json-schema.org/keyword/type"]) {
+    let allowedTypes = new Set(ALL_TYPES);
```

Comment:
There's no reason to create a new set every time you initialize `allowedTypes`. `ALL_TYPES` can be a set.